<html>
<head> <style>  <!-- /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
{margin:0cm; margin-bottom:.0001pt; font-size:12.0pt; font-family:"Times; New Roman";}
p.MsoPlainText, li.MsoPlainText, div.MsoPlainText
{margin:0cm; margin-bottom:.0001pt; font-size:10.0pt; font-family:"Courier New";}
@page Section1  {size:595.3pt 841.9pt; margin:72.0pt 57.6pt 72.0pt 57.6pt;}
div.Section1    {page:Section1;}
-->  </style> </head>
<body lang=EN-US>  <div class=Section1>
<p class=MsoPlainText>Mapping plant interactome networks</p>
<p class=MsoPlainText>&nbsp;</p>
<p class=MsoPlainText>ANNE-RUXANDRA CARVUNIS</p>
<p class=MsoPlainText>DANA FARBER CANCER INSTITUTE</p>
<p class=MsoPlainText>&nbsp;</p>

<p class=MsoPlainText>Protein-protein interactions are an essential constituent of all cells and organisms. Proteome-scale protein-protein interaction maps, or “interactome maps”, have become an important source of information for biology. Interactome network maps contain countless hypotheses that can assist small-scale biological studies focused on one or a few proteins at-a-time. They also provide network information for topological analysis and help the development of large-scale dynamic models. Lastly, protein interaction information has been successfully combined with other types of large-scale data to further our understanding of pathways and biological networks. Arabidopsis thaliana is one of the best-studied plant model organisms for which crucial genome-wide resources are available. Furthermore, the recent completion of sequencing projects for several plant genomes has revealed an enormous amount of conservation among their genomes. The Center for Cancer Systems Biology (CCSB - Boston) and the Salk Institute Genomic Analysis Laboratory (SIGnAL - San Diego) have started a collaboration to systematically map plant interactome networks by exploiting this high level of conservation. Making use of a genome-scale Arabidopsis thaliana protein-coding open reading frame collection (ORFeome) containing 14,000 ORF reagents generated by SIGnAL, we are employing high throughput, high-quality yeast-2-hybrid and protein array technology to create a first generation plant interactome map. This effort will be complemented by mapping experiments for selected rice (Oryza sativa) gene-products. CCSB has developed a high throughput, robust platform for binary protein-protein interaction mapping and has published first draft maps for binary interactomes of H. sapiens and C. elegans and a second generation map of S. cerevisiae (Rual et al., Nature 2005; Walhout et al., Science 2000; Li et al., Science 2004, Yu et al., Science 2008). Importantly, we have implemented quality control mechanisms that can provide high confidence, validated binary interaction data for any interactome mapping project (Venkatesan et al., Nature Methods 2009; Simonis et al., Nature Methods 2009; Braun et al., Nature Methods 2009; Cusick et al., Nature Methods 2009). Here we will present a first network analysis of the completely mapped ‘Space-1’ of the Arabidopsis thaliana interactome and a comparative analysis with previously mapped protein networks of yeast, worm and man.</p>
</div>
</body>
</html>
