<html>
<head> <style>  <!-- /* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
{margin:0cm; margin-bottom:.0001pt; font-size:12.0pt; font-family:"Times; New Roman";}
p.MsoPlainText, li.MsoPlainText, div.MsoPlainText
{margin:0cm; margin-bottom:.0001pt; font-size:10.0pt; font-family:"Courier New";}
@page Section1  {size:595.3pt 841.9pt; margin:72.0pt 57.6pt 72.0pt 57.6pt;}
div.Section1    {page:Section1;}
-->  </style> </head>
<body lang=EN-US>  <div class=Section1>
<p class=MsoPlainText>Towards more biologically realistic update functions in real-life regulatory network models</p>
<p class=MsoPlainText>&nbsp;</p>
<p class=MsoPlainText>CHRISTIAN DARABOS</p>
<p class=MsoPlainText>UNIVERSITY OF LAUSANNE</p>
<p class=MsoPlainText>&nbsp;</p>

<p class=MsoPlainText>It has been suggested that the cells of living organisms are functioning in a near-chaotic regime called critical, which offers a trade-off between stability and evolvability. Abstract models for genetic regulatory networks (GRNs) such as Kauffman’s Random Boolean Networks (RBNs) [4] certainly point in that direction. Although very attractive for its simplicity, the original RBN model suffers a number of limitations that can be overcome by incorporating recent discoveries. We addressed some of these weaknesses in a previous work [2], namely the synchronicity and the random gene interaction, in a previous work. Here, we tackle different aspects of the RBN’s original shortcomings. Firstly, even if their exact values are unknown, it is clear that gene update functions should not be random. Gene expression rests on the combined effect of incoming proteins that can have a promoting or repressing action on their target genes. Stoll et al. [7] have proposed a simple additive dynamical rule that characterizes the temporal evolution of the gene’s state. They consider that both the activating and repressing factors have the same weight, and thus, the state of a target gene at the next time-step Si(t+1) will be: active if it receives a majority of promoting components from already active genes, inactive it receives a majority of repressing components, or the state of the target gene will remain unchanged in case the number of promoting and repressing inputs are equal. Inspired by their work, we propose an update function shared by all genes that takes into account the fact that promoting and repressing components could have asymmetrical effects. A gene could require a majority by more than one active gene to switch states, and therefore, we introduced a threshold parameter which has to be met in order for a gene to activate. Moreover, as some gene of our model might not have any repressors, and, if activated, should not remain in this state permanently, we add a decay component. In the case where an active gene has no repressor at all, we switch it to inactive manually at the next time-step. This update function is equivalent to Stoll’s in the case where the threshold value is 0.5. Additionally, all rules in this class can be proven to correspond to a subset of the original RBN update functions. This offers the advantage of making the new model state space exhaustively enumerable. Another questionable assumption of the original RBNs model is the totally random interaction among genes with a fixed connectivity. Thanks to the recent developments in high throughput genomic and bio- chemical techniques, small parts of real-life GRNs have been discovered with data sufficiently reliable to specify highly probable interaction, with different confidence rates. We have selected the core transcriptional network in embryonic cells published by Chen et al. [1] and a portion of the yeast cell-cycle by Li et al. [5] as a substrate for our RBN model. Original RBNs go through a phase transition by tuning parameters such as the degree of the nodes and probability of a gene to be expressed. Considering current knowledge about GRNs, some of Kauffman’s properties of the model are now subject to criticism. In our case, we use real-life networks and not artificial ones, and thus we cannot tune any property of the network topology to obtain the desired critical regime. Instead, we can adjust the threshold value in the update function to set the system into a critical regime. We propose a new measurement of the closeness to the critical regime: the criticality distance. This metric uses a dynamical property of the whole system, usually visualized with Derrida plots [3], and offers a way of discriminating the regime in which RBN-like dynamical systems evolve. The criticality distance quantifies the convergence versus the divergence in the state space of a RBN. It can in turn characterize the different regimes, using the Hamming distance to compute the Euclidian distance between the RBN dynamics and the main diagonal typical of Derrida plots. Other possible ways of determining the actual regime of a system would require either scaling of the systems’ size or analyzing the systems’ response to failure, but neither is an option in the present work. Finally, to validate our model we use a third dynamic Boolean sub-network from plant biology presented by Li et al. [6]. In this model the actual Boolean function of each component in the network has been established and our analysis clearly shows it operates in the critical regime. The results are excellent, and prove that in this particular case, our Boolean functions are much closer to biological ones than random ones with an overlap of approximately 92%. Taking into account recent years’ advances in the field of cellular biology, we have proposed to identify under what conditions Kauffman’s hypothesis that living organism cells operate in a region bordering order and chaos holds. This property confers to living organisms both the stability to resist transcriptional errors and external disruptions, and, at the same time, the flexibility necessary to evolution. We studied two particular cases of genetic regulatory networks found in literature in terms of complex dynamical systems derived from the original RBN model. In order to do that, we compared the behavior of these systems under the original update function and the novel additive function that we believe is closer to the actual role of living organisms. We successfully identify the conditions in which our model’s response can be interpreted as critical, thus most biologically plausible. Results of numerical simulations show that there exist values in both update functions that allow the models to operate in the critical region, and that these values are comparable in two different real-life GRNs. Acknowledgements: The authors thank in particular Réka Albert and her research associates for useful discussions of the network used to validate the proposed model. M. Tomassini and Ch. Darabos gratefully acknowledge financial support by the Swiss National Science Foundation under contract 200021-107419/1, and by the Rectors’ Conference of the Swiss Universities. M. Giacobini and P. Provero acknowledge funding (60% grant) by the Ministero dell’Universit`a e della Ricerca Scientifica e Tecnologica. F. Di Cunto and M. Giacobini acknowledge Compagnia di San Paolo financial support. References [1] Xi Chen, Han Xu, Ping Yuan, Fang Fang, Mikael Huss, Vinsensius B. Vega, Eleanor Wong, Yuriy L. Orlov, Weiwei Zhang, Jianming Jiang, Yuin-Han Loh, Hock Chuan Yeo, Zhen Xuan Yeo, Vipin Narang, Kunde Ramamoorthy Govindarajan, Bernard Leong, Atif Shahab, Yijun Ruan, Guillaume Bourque, Wing-Kin Sung, Neil D. Clarke, Chia-Lin Wei, and Huck-Hui Ng. Integration of external signaling pathways with the core transcriptional network in embryonic stem cells. Cell, 133(6):1106– 1117, 06 2008. [2] Ch. Darabos, M. Tomassini, and M. Giacobini. Dynamics of unperturbed and noisy generalized boolean networks. Journal of Theoretical Biology, 260(4):531 – 544, 2009. [3] B. Derrida and Y. Pomeau. Random networks of automata: a simple annealed approximation. Europhysics Letters, 1(2):45–49, 1986. [4] S. A. Kauffman. Metabolic stability and epigenesis in randomly constructed genetic nets. J. Theor. Biol., 22:437–467, 1969. [5] Fangting Li, Tao Long, Ying Lu, Qi Ouyang, and Chao Tang. The yeast cell-cycle network is ro- bustly designed. Proceedings of the National Academy of Sciences of the United States of America, 101(14):4781–4786, 2004. [6] Song Li, Sarah M Assmann, and R ?eka Albert. Predicting essential components of signal transduction networks: A dynamic model of guard cell abscisic acid signaling. PLoS Biol, 4(10):e312, 09 2006. [7] Gautier Stoll, Jacques Rougemont, and Felix Naef. Representing perturbed dynamics in biological network models. Phys. Rev. Lett. E, 76, 2007.</p>
</div>
</body>
</html>
