
* An edge that excites or inhibits another edge (instead of a node)
seen on: http://cneuro.rmki.kfki.hu/sites/default/files/nnw6.pdf

* Probabilistic causal networks, such as Boolean, Bayesian, and probabilistic
Gaussian networks, include edges with direction, and therefore can represent
causal relationships among genes when causality is known. The inclusion of this
information can lead to improved predictions of response to various perturbation
events [19,20]. Recently, significant research interest has shifted to the use of
Bayesian networks to study causal interaction networks of biological systems based
on gene expression data from time series and gene knockout experiments, protein–protein
    interaction data derived from predicted genomics features, and other direct experimental interaction data [3,21].
http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.0030069

* dynamic Bayesian networks [31], which explicitly allow for a temporal representation of how nodes in the network interact with one another.
Rabiner LR (1989) A tutorial on Hidden Markov Models and selected applications in speech recognition. Proceedings of the 77th Meeting of the Institute of Electrical and Electronics Engineers, 15–18 May 1989; San Diego, California, United States. IEEE 77: 257–286.

* provenance ontology that might have some insight for modeling dynamic networks' nodes and edges
http://wiki.knoesis.org/index.php/Provenir_Ontology
